pkgs/package-overrides.nix

```diff
@@ -112,6 +112,18 @@ in
       else
         prev.extensions.memcached;
 
+    mysql = lib.mkIf (lib.versionOlder prev.php.version "7.0") (prev.mkExtension {
```

Comment:
`mkIf` is for the module system, it will not work here. You need `if … then … else … null`.

Reply:
ha ha ha... oh boy. Yeah, it really really is :smile: